students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_1/table_printer.py

```diff
@@ -0,0 +1,18 @@
+def print_table(table):
+    columns_quantity = len(table)
+    rows_quantity = len(table[0])
+    column_widths = [0] * columns_quantity
+    for column in range(columns_quantity):
+        column_widths[column] = len(max(table[column], key=len))
```

Comment:
you don't really need to preallocate the `column_widths`
```
column_widths = [len(max(column, key=len)) for column in table]
```

also `map` can be used to produce list of lengths and pick max without extra keying
```
column_widths = [max(map(len, column)) for column in table]
```